packages/flutter_tools/templates/app/linux.tmpl/app.desktop.tmpl

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name={{projectName}}
+Comment={{description}}
+Icon=icon.png
+Exec={{projectName}} %U
+Terminal=false
+Type=Application
+Categories=Other;
+Keywords=Other;
```

Comment:
Maybe put a comment on these two lines explaining what they are and where to get more information, since we expect people to change them presumably. We should assume most people won't be familiar with these files and provide help and/or links to instructions.